src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/choose-your-aggregation-method.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Choose your aggregation method
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+metaDescription: "Use this for guidance on choosing the best aggregation method for your alerting conditions and notification thresholds."
+---
+
+Alerts conditions provide a sophisticated set of tools for describing when you want to be notified about something that's happened or failed to happen on something you're monitoring. For best results, choose your aggregation method that best matches the way your data arrives.
+
+The three aggregation methods are event flow, event timer, and cadence. If you're interested in a conceptual overview, see our doc on [streaming alerts, key terms and concepts](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/streaming-alerts-key-terms-concepts/).
+
+## What's aggregation? [#aggregation]
+
+When an application or service is monitored by New Relic, data can arrive in different ways. Some data arrives consistently and predictably, while other data arrives inconsistently and sporadically.
+
+Aggregation is how our alerting system gathers data together before analyzing it for exceeding warning or critical threshold levels. 
+
+Your data is collected as data points in an aggregation window and then turned into a single numeric value. The data points are aggregated based on your NRQL query using methods like sum, average, min, and max, among others. This single numeric value is what's used to evaluate the condition's threshold.
+
+Once data has been aggregated, no more data points can be added to it. Our different aggregation methods will help you strike a balance between aggregating your data quickly and waiting for enough data points to arrive. 
+
+## Why it matters [#why]
+
+Choosing the right aggregation method is an important step toward calibrating your alerting system so that you get notifications for events that are important to you while preventing unwanted notifications.
+
+The most important questions to consider when deciding on your aggregation method: How often does my data arrive? How consistently does my data arrive?
```

Comment:
How do these questions answer which aggregation method to use? Maybe adding an example could help.

Reply:
I brought the brief aggregation method sentences up from their sections, because I think they're more valuable here.